biomage/unstage/unstage.py

```diff
@@ -31,16 +204,30 @@ def check_if_exists(org, sandboxid):
     default="biomage-ltd",
     help="The GitHub organization to perform the operation in.",
 )
-def unstage(token, org, sandboxid):
+@click.option(
+    "--resources-only",
```

Comment:
I think this flag is not needed. The unstage should try to remove the CI resources and if they don't exist then just fail silently and then proceed to remove the data.

Reply:
Hmmm.. ok, rearranged code to give notification and information proceeding to removal of resources